demo-angular/package.json

```diff
@@ -30,16 +30,21 @@
   },
   "devDependencies": {
     "htmlparser2": "^3.9.2",
+    "@angular/compiler-cli": "~6.0.0",
```

Comment:
I think the changes to this file are not needed and it should not be committed.

Reply:
Reverted to original but added the `@angular/compiler-cli` and `@ngtools/webpack` `devDependencies` that are automatically added by the new webpack